### DISCUSSION TITLE
Why built file size more than 40 mb?

### DISCUSSION THREAD

**Discussion by chteeraphong** [keycloakify/keycloakify#373] [General]
Hi everyone,
I found an issue with the built file size, which is larger than 40 MB, ~ 80 MB.
I cannot upload to cloud-iam, I wonder why the file size increased from ~40 MB to ~80 MB.

**Comment by garronej** [comment]
Hello @chteeraphong,  

I've answered in [this discution](https://github.com/keycloakify/keycloakify/issues/367) but if you reach some limitation for the upload of your jar I guess it's woth investigating what file I cand avoid undling to reduce the size.  

Quick win: to divide by 2 your bundle size you can remove https://github.com/keycloakify/keycloakify-starter/blob/16843937baa512d75f2a0f6b0aa8ec489e985046/package.json#L21-L23 from your package.json (if you have it)
